clientv3/namespace/kv.go

```diff
@@ -48,7 +48,7 @@ func (kv *kvPrefix) Put(ctx context.Context, key, val string, opts ...clientv3.O
 }
 
 func (kv *kvPrefix) Get(ctx context.Context, key string, opts ...clientv3.OpOption) (*clientv3.GetResponse, error) {
-	if len(key) == 0 {
+	if len(key) == 0 && !clientv3.IsWithFromKey(opts) {
```

Comment:
@cfc4n also, I think there is a similar problem with WithPrefix so it would be good to fix it here as well adding covering both for get and delete -  `!(clientv3.IsWithFromKey(opts) || clientv3.IsWithPrefix(opts))`